Unittests fail because Unable to locate package python3-mans-to-es

```E: Unable to locate package python3-mans-to-es
Error: Process completed with exit code 100.

```
https://github.com/google/timesketch/blob/cb62b592ad36e33f54b9a3399b147827ddcfe329/.github/workflows/unit-tests.yml#L59

Need to check if we still need that package.

--- a/.github/workflows/unit-tests.yml
+++ b/.github/workflows/unit-tests.yml
@@ -56,7 +56,7 @@ jobs:
             python3-flask-restful python3-flask-script python3-flask-sqlalchemy \
             python3-flaskext.wtf python3-google-auth python3-google-auth-oauthlib \
             python3-gunicorn python3-idna python3-itsdangerous python3-jinja2 \
-            python3-jsonschema python3-jwt python3-kombu python3-mako python3-mans-to-es \
+            python3-jsonschema python3-jwt python3-kombu python3-mako \
             python3-markdown python3-markupsafe python3-neo4jrestclient python3-numpy \
             python3-oauthlib python3-pandas python3-parameterized python3-pycparser \
             python3-pyrsistent python3-redis python3-requests python3-requests-oauthlib \

--- a/data/timesketch.conf
+++ b/data/timesketch.conf
@@ -144,13 +144,13 @@ GOOGLE_OIDC_HOSTED_DOMAIN = None
 GOOGLE_OIDC_ALLOWED_USERS = []
 
 #-------------------------------------------------------------------------------
-# Upload and processing of Plaso storage or mans files.
+# Upload and processing of Plaso storage files.
 
 # To enable this feature you need to configure an upload directory and
 # how to reach the Redis database used by the distributed task queue.
 UPLOAD_ENABLED = False
 
-# Folder for temporarily storage of Plaso dump or mans files before being processed and
+# Folder for temporarily storage of Plaso dump files before being processed and
 # inserted into the datastore.
 UPLOAD_FOLDER = '/tmp'
 

--- a/importer_client/python/timesketch_import_client/importer.py
+++ b/importer_client/python/timesketch_import_client/importer.py
@@ -629,7 +629,7 @@ class ImportStreamer(object):
 
         else:
             raise TypeError(
-                'File needs to have a file extension of: .csv, .jsonl, mans or '
+                'File needs to have a file extension of: .csv, .jsonl or '
                 '.plaso')
 
     def add_json(self, json_entry, column_names=None):

--- a/timesketch/lib/tasks.py
+++ b/timesketch/lib/tasks.py
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Celery task for processing Plaso storage or mans files."""
+"""Celery task for processing Plaso storage files."""
 
 from __future__ import unicode_literals
 

